File: src/main/java/com/linebot/entity/Request.java
package com.linebot.entity;

import lombok.*;

import javax.persistence.*;
import java.util.Date;

@Builder
@Data
@Entity
@Table(name = "request")
@NoArgsConstructor
@AllArgsConstructor
public class Request {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;
    @Column(name = "user_id")
    private String userId;
    @Column(name = "request")
    private String request;
    @Column(name = "status")
    private int status;
    @Column(name = "deleted")
    private String deleted;
    @Column(name = "created_at")
    private Date createdAt;

}

File: src/main/kotlin/com/linebot/entity/Request.kt
package com.linebot.entity

import java.util.*
import javax.persistence.*

@Entity
@Table(name = "request")
data class Request(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        @Column(name = "id")
        var id: Int? = null,
        @Column(name = "user_id")
        var userId: String? = null,
        @Column(name = "request")
        var request: String? = null,
        @Column(name = "status")
        var status: Int = 0,
        @Column(name = "deleted")
        var deleted: String? = null,
        @Column(name = "created_at")
        var createdAt: Date? = null
)
